docs/en/week06/06-3.md

```diff
@@ -0,0 +1,203 @@
+
+# Deep Learning Lab 6: RNN 
+
+Authors:
+    - Zhengyuan Ding
+    - Biao Huang
+    - Lin Jiang
+Editor: 
+    - Nhung Le
```

Comment:
What is this??? Check other submissions for the format.
See https://raw.githubusercontent.com/Atcold/pytorch-Deep-Learning/master/docs/en/week05/05-3.md